ofxProjectGenerator/src/addons/ofAddon.cpp

```diff
@@ -261,10 +261,15 @@ void ofAddon::parseVariableValue(string variable, string value, bool addToValue,
 		addReplaceStringVector(includePaths,value,addonRelPath,addToValue);
 	}
 
+	
 	if(variable == "ADDON_CFLAGS"){
 		addReplaceStringVector(cflags,value,"",addToValue);
 	}
 
+	if (variable == "ADDON_PREPROCESSOR_DEFINITIONS") {
```

Comment:
this reference to ADDON_PREPROCESSOR_DEFINITIONS should go as well

Reply:
so, then adding ADDON_PREPROCESSOR_DEFINITIONS is useful?

Since I changed 
`additionalOptions = items[i].node().child("ClCompile").child("AdditionalOptions");`
for
`additionalOptions = items[i].node().child("ClCompile").child("PreprocessorDefinitions");`

Should we leave it like this or should we use the seperate preprocessor flag?

Reply:
no sorry, what i meant is it should go away

Reply:
already is in the latest commit